vmm/src/vm.rs

```diff
@@ -875,6 +877,21 @@ impl Vm {
             None,
         ) {
             Ok(entry_addr) => entry_addr,
+            // Try to load the binary as kernel PE file at first.
```

Comment:
This is related to a `linux-loader` proposal https://github.com/rust-vmm/linux-loader/pull/75.

To load the format-less UEFI binary, we figured out this logic.
@rbradford @sboeuf could you please take a look and give some comments?

Reply:
Added the load function in `arch/`.